plugin/aws/ec2Instance.go

```diff
@@ -66,6 +68,38 @@ func newEC2Instance(ctx context.Context, inst *ec2Client.Instance, session *sess
 	return ec2Instance
 }
 
+func (inst *ec2Instance) cachedConsoleOutput(ctx context.Context, latest bool) (consoleOutput, error) {
+	output, err := plugin.CachedOp(ctx, "ConsoleOutput", inst, 30*time.Second, func() (interface{}, error) {
```

Comment:
Oh arg, `latest` now means two calls to `CachedOp` may have different inputs. Can you use different strings when latest is true vs false? Something like `ConsoleOutput` vs `ConsoleOutputLatest`?

Reply:
This is probably more complicated than it needs to be now. I can try to do some extra cleanup later.

Reply:
Sure, I think I know what you mean. No worries, I'll have a look in a bit.

Reply:
Totally fine if you squash commits btw.

Reply:
Whoops, just saw your last post. Yes, that sounds good.

Reply:
Can you still push the change for using different cache prefixes: ConsoleOutput and ConsoleOutputLatest?

Reply:
Well, this line of code...

```output, err := plugin.CachedOp(ctx, "ConsoleOutput", inst, 30*time.Second, func() (interface{}, error) { ... ```

involves (I think) advanced concepts that I don't have a deep enough understanding of yet., therefore it might best for you to provide the code needed in this case. Or... is it just a matter of doing something like this?

``` 
func (inst *ec2Instance) cachedConsoleOutput(ctx context.Context, latest bool) (consoleOutput, error) {
  var opname string
  if latest {
    opname = "ConsoleOutputLatest"
  } else {
    opname = "ConsoleOutput"
  }
  output, err := plugin.CachedOp(ctx, opname, inst, 30*time.Second, func() (interface{}, error) { ... 
```

Reply:
Just what you proposed is what I had in mind.